python/ray/_raylet.pyx

```diff
@@ -980,12 +971,14 @@ cdef class CoreWorker:
             return VectorToObjectIDs(return_ids)
 
     def create_actor(self,
-                     function_descriptor,
+                     Language language,
```

Comment:
```suggestion
                     Language target_language,
```

Reply:
I think we can keep using `language` for now and switch to `target_language` if we have `source_language`.